slides/src/content/a11y.md

```diff
@@ -218,20 +218,17 @@ For example in your header div:
 ## Accessible Development - Tables
 
 - Layout tables should be done using CSS and DIVS
-  - Include ARIA roles to define table elements for screen readers
+  - Include ARIA roles to define table elements for screen readers if using divs
 - If developing layout tables in HTML
   - Use the non-standard `datatable=0` attribute on the Table tags to tell JAWS to ignore layout tables
   - Don't use the `summary` attribute on the Table tag if you must create a layout table (or set the `summary` attribute to null
   - Don't use `<TH>` or Caption tags on layout tables
   - You can use `<TH>` and summary attributes for data tables and include the `scope="ROW"` or `scope="COL"` attributes to indicate what     the `<TH>` is for
 
-
-
 ---
 <!-- .slide: id="a11y" -->
-## Accessible Development - Example CSS Layout Table
+## Accessible Development - Example CSS - Layout Table for Divs
```

Comment:
This title will line-break in slide mode - just use "Layout Table for Divs" as slide title?